Assets/ScriptableObjects/Plugin Scripts/MoveablePlugin.cs

```diff
@@ -32,5 +31,7 @@ public override void OnFaceClick(BlockFace face)
                 _displacedFace = face;
             }
         }
+
+        _block.ResetSpeed();

```

Comment:
I don't think move block should explicitly reset the speed.

Reply:
Instead could you have `MoveBlock` take in an initial speed that lets the plugin explicitly set what initial speed he wants. To make this even cleaner we can have a optional parameter for it. https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments